assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -23,6 +23,25 @@ function APIProvider( { children } ) {
 		[ stories ],
 	);
 
+	const saveStoryById = useCallback(
+		( storyId, title, status, pages, author, slug ) => {
```

Comment:
To clarify, is the `pages` here an array of separated story pages? And all the story-wide properties would go as separate props here? Is there a reason not to put them all together into a single `story` JSON structure?

Reply:
`Pages` here is a array of object that represents the pages in the story. The other fields are just WordPress fields.

Reply:
I'd think that we should probably make `pages` a field in a separate `story` object. There will definitely be some story wide properties.

Reply:
Agreed. A `story` object makes sense to me 👍

Reply:
A story is a group of separate fields. I am still not sure what the benefit of a story object here.

Reply:
See https://github.com/ampproject/amp-wp/pull/3871#issuecomment-565461091

Reply:
Marking as resolved, as all questions of state are well out of scope of this PR. Let's move this conversion to #3910